tests/data/doc_write/rdf-mini.json

```diff
@@ -3,7 +3,7 @@
     "@xmlns:ns1": "http://spdx.org/rdf/terms#",
     "@xmlns:rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
     "ns1:SpdxDocument": {
-      "@rdf:about": "http://www.spdx.org/tools#SPDXRef-DOCUMENT",
+      "@rdf:about": "None#SPDXRef-DOCUMENT",
```

Comment:
`None` is not a valid namespace, I believe... Why not use the one that the rest of the test files use?

Reply:
This test is based on the function `_get_mini_doc()` in test_document.py. This minimal document seems to be from some earlier SPDX specification where the namespace probably wasn't mandatory. I can add the namespace to the function as a hoepfully minimal-invasive fix..

Reply:
Nah, if that is some special, long forgotten case, I don't think we should bother.